kalagraphs.module

```diff
@@ -286,3 +299,57 @@ function _kalagraphs_process_form($subform, FormStateInterface &$form_state, &$c
 function kalagraphs_entity_type_alter(array &$entity_types) {
   $entity_types['paragraph']->setClass('Drupal\kalagraphs\Entity\Kalagraph');
 }
+
+/**
+ * Prepares variables for theme_inline_entity_form_entity_table().
+ *
+ * Override bricks_inline_preprocess_inline_entity_form_entity_table() so we can
+ * customize the tabledrag table.
+ */
+function kalagraphs_preprocess_inline_entity_form_entity_table(&$variables) {
```

Comment:
This code looks great, thanks. I think I'll move my custom code from `Kalagraphs.php` (where I override the entity label to insert the field info) to here instead.